lib/events/auditlog_test.go

```diff
@@ -110,53 +69,66 @@ func (a *AuditTestSuite) TestLogRotation(c *check.C) {
 			Metadata:     events.Metadata{Type: "resize", Time: now},
 			TerminalSize: "10:10",
 		}
-		err = alog.EmitAuditEvent(context.TODO(), event)
-		c.Assert(err, check.IsNil)
+		err := alog.EmitAuditEvent(context.TODO(), event)
+		require.NoError(t, err)
 		logfile := alog.localLog.file.Name()
 
 		// make sure that file has the same date as the event
 		dt, err := parseFileTime(filepath.Base(logfile))
-		c.Assert(err, check.IsNil)
-		c.Assert(dt, check.Equals, time.Date(now.Year(), now.Month(), now.Day(), 0, 0, 0, 0, now.Location()))
+		require.NoError(t, err)
+		require.Equal(t, dt, time.Date(now.Year(), now.Month(), now.Day(), 0, 0, 0, 0, now.Location()))
```

Comment:
Technically the order of the expected/actual values in the assertion is wrong, which is fine other than the confusing error message you get if things fail.